README.md

```diff
@@ -1,2 +1,50 @@
-# foreman-operations-collection
-Ansible collection for management of Foreman servers
+# Foreman Operations Collection ![Build Status](https://github.com/theforeman/foreman-collection-operations/workflows/CI/badge.svg)
+
+Ansible assets for managing Foreman operations such as install, upgrade or maintenance tasks.
+
+## Documentation
+
+A list of all assets and their documentation can be found at [theforeman.org/plugins/foreman-operations-collection](https://theforeman.org/plugins/foreman-operations-collection/).
+
+## Support
+
+### Supported Foreman and plugins versions
+
+Ansible assets should support any currently stable Foreman release and the matching set of plugins.
+Some modules have additional features/arguments that are only applied when the corresponding plugin is installed.
+
+We actively test the modules against the latest stable Foreman release and the matching set of plugins.
+
+### Supported Ansible Versions
+
+The supported Ansible versions are aligned with currently maintained Ansible versions. You can find the list of maintained Ansible versions [here](https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#release-status). Older Ansible versions (Ansible >= 2.3) should work but are not tested.
+
+As we're using Ansible's [documentation fragment](https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#documentation-fragments) feature, that was introduced in Ansible 2.8, `ansible-doc` prior to 2.8 won't be able to display the module documentation, but the modules will still run fine with `ansible` and `ansible-playbook`.
+
+### Supported Python Versions
+
+Starting with Ansible 2.7, Ansible only supports Python 2.7 and 3.5 (and higher). These are also the only Python versions we develop and test the modules against.
```

Comment:
2.7 (Ansible) is definitely not supported anymore...

Reply:
Agreed, I do not think this is intended to say that. I think is this just giving history that in Ansible 2.7 they started only supporting Python 2.7 and 3.5 (or higher). That is, putting the blame on Ansible. I can just reword this :)